tests/helpers/test_device_registry.py

```diff
@@ -428,7 +446,7 @@ async def test_migration_1_2_to_1_3(hass, hass_storage):
         config_entry_id="1234",
         connections={("Zigbee", "01.23.45.67.89")},
         identifiers={("serial", "12:34:56:AB:CD:EF")},
-        hw_version="new_version",
+        sw_version="new_version",
```

Comment:
Was this incorrect?

Reply:
If we want to trigger a write, without modifying the migrated data, it was previously incorrect.

Reply:
Ok, I see what the test wants to do now.